bluesky.py

```diff
@@ -62,6 +62,15 @@ def format_for_source_url(self, id):
     """
     return quote(quote(id, safe=''))
 
+  def post_id(self, url):
+    if url.startswith('at://'):
+      if url.startswith(f'at://{self.username}'):
+        # Bluesky can't currently resolve AT URIs containing handles,
+        # even though they are technically valid. Replace it with DID.
+        return url.replace(self.username, self.key_id())
```

Comment:
mind switching to something like this?

```suggestion
        return url.replace(f'at://{self.username}', f'{at://self.key_id()}')
```

It's unlikely, but if someone ever uses a handle that also shows up in a collection NSID, eg `bsky.feed` and the `app.bsky.feed.post` collection, this will make sure we don't modify the collection too.